packages/admin/cms-admin/src/generator/future/generateForm.ts

```diff
@@ -58,6 +58,8 @@ export function generateForm(
         \${${`${instanceGqlType}FormFragment`}}
     `;
 
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore TS2589: Type instantiation is excessively deep and possibly infinite.
```

Comment:
this hint is true, as we use recursion to create fields. but this is required. I didn't look up a different solution or a workaround for this typescript-error.

Reply:
this is not a hint, its an error :)

And it says the types are recursive (not the runtime code) which you should avoid.

Reply:
alright, then I will look into it 👍

Reply:
solved 👍